jupyterlite_sphinx/_try_examples.py

```diff
@@ -3,13 +3,17 @@
 import re
 
 
-def examples_to_notebook(input_lines):
+def examples_to_notebook(input_lines, warning_text=None):
```

Comment:
Make it keyword only for simple future API changes ? 

```suggestion
def examples_to_notebook(input_lines, *, warning_text=None):
```